### PR TITLE
feat(ffi): add an error kind to PickyError

### PR DESCRIPTION
**Pull request by CBenoit**
Useful when programmatically matching on the underlying error cause.

cc @MathieuMorrissette

### CODE REVIEW

File: ffi/dotnet/Devolutions.Picky/Generated/PickyError.cs [hunk 1]
```diff
@@ -12,12 +12,20 @@ namespace Devolutions.Picky;
 #nullable enable
 
 /// <summary>
-/// Stringified Picky error.
+/// Stringified Picky error along an error kind
```
Comment (thenextman) [inline ffi/dotnet/Devolutions.Picky/Generated/PickyError.cs:15]: nit: along _with_ an error kind?

**Review by thenextman — APPROVED** [review]
LGTM

**Review by MathieuMorrissette — APPROVED** [review]
LGTM

File: ffi/dotnet/Devolutions.Picky/Generated/PickyError.cs (hunk 1 above)
Comment (CBenoit) [reply ffi/dotnet/Devolutions.Picky/Generated/PickyError.cs:15]: Thank you, I fixed that!

**Pull request merged**